hummingbot/strategy/cross_exchange_market_making/order_id_market_pair_tracker.pyx

```diff
@@ -27,6 +30,9 @@ cdef class OrderIDMarketPairTracker(TimeIterator):
         TimeIterator.c_tick(self, timestamp)
         self.c_check_and_expire_tracking_items()
 
+    def tick(self, timestamp: float):
+        self.c_tick(timestamp)
```

Comment:
This should be removed because the same method is already implemented in the superclass

Reply:
✅